Gemfile

```diff
@@ -33,22 +34,24 @@ gem 'jbuilder', '~> 2.5'
 # Use Capistrano for deployment
 # gem 'capistrano-rails', group: :development
 
+gem 'rubocop', require: false
```

Comment:
please put this line at `group :development, :test`, we use this gem just for development and test environment

Reply:
Done.. only missing push it